sem_covid/entrypoints/etl_dags/eurlex_covid19.py

```diff
@@ -384,14 +543,32 @@ def upload_processed_documents_to_elasticsearch():
         try:
             logger.info(f'Sending to ElasticSearch ( {config.EU_CELLAR_IDX} ) the object {obj.object_name}')
             es_adapter.index(index_name=config.EU_CELLAR_IDX, document_id=obj.object_name.split("/")[1],
-                                       document_body=loads(minio.get_object(obj.object_name).decode('utf-8')))
+                             document_body=loads(minio.get_object(obj.object_name).decode('utf-8')))
             object_count += 1
         except Exception as ex:
             logger.exception(ex)
 
     logger.info(f'Sent {object_count} file(s) to ElasticSearch.')
 
 
+def get_items():
```

Comment:
this is where I was asking myself if we could spawn a DAG step (operator) for each of the single items that are being downloaded. Then we have granular control within the DAG over each individual download. 
This was possible by embedding these multiple PythonOperators into an operator group. We have a story about this in JIRA.